<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QWidget" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connections</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="connectionTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
